README.md

```diff
@@ -72,7 +72,7 @@ JKQ QMAP is developed as a C++ library with an easy to use Python interface.
     ```
   However, in order to get the best performance out of QMAP, it is recommended to build it locally from the source distribution (see [system requirements](#system-requirements)) via
     ```bash
-    pip install --no-binary jkq.qmap
+    pip install --no-binary jkq.qmap jkq.qmap
```

Comment:
```suggestion
    pip install  jkq.qmap --no-binary jkq.qmap
```
Wouldn't it be more intuitive to write it in this fashion?

Reply:
right :)